components/events/EventCard.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <div class="card">
+        <div class="title_block">
+            <div class="title">{{ title }}</div>
+            <div class="tag" :class="[tagColor]">{{ tag }}</div>
+        </div>
+        <p class="card_txt">
+            {{ description }}
+        </p>
+        <div class="card_img">
+            <img :src="imgUrl" alt="imgAlt" />
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'EventCard',
+    props: {
+        title: { type: String, default: '' },
+        description: { type: String, default: '' },
+        tag: { type: String, default: '' },
+        tagColor: { type: String, default: '' }, // red, blue, orange, green
+        imgUrl: { type: String, default: '' },
+        imgAlt: { type: String, default: '' },
+    },
+}
+</script>
+
+<style lang="css" scoped>
+.card {
+    width: 380px;
+    min-height: 486px;
+    border-radius: 24px;
+    border: 2px solid #c2a53a;
+    box-shadow: 6px 6px 0px #c2a53a;
+    padding: 34px 30px 38px;
+    margin: 0px 22px 44px;
+}
+.card:hover {
+    background-color: #e6ba17;
+}
+
+.title_block {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-bottom: 24px;
+}
+.card:hover .title {
+    color: #000;
+}
+.title {
+    font-weight: bold;
+    font-size: 24px;
+    line-height: 30px;
+    color: #c2a53a;
+}
+.card:hover .tag {
+    color: #000;
+    border: 2px solid #000;
+}
+.tag {
+    border-radius: 8px;
+    font-weight: 600;
+    font-size: 12px;
+    padding: 6.5px 14px;
```

Comment:
How about just set this to `7px`? As float point pixels may render differently among browsers.

Reply:
fixed